tbtaf/tbtaf_launcher.py

```diff
@@ -7,15 +7,28 @@
 from __future__ import print_function
 from orchestrator.TBTAFOrchestrator import TBTAFOrchestrator
 from listener.TBTAFListener import TBTAFListener
+from interpreter.TBATFInterpreterCLI import TBTAFInterpreterCLI
+import sys
 
 if __name__ == '__main__':
-    testScript = './test/test04.tbtaf'
-    myTBTAF = TBTAFOrchestrator()
+    ubicacion = './test/temp.tbtaf'
+    comando = ""
     
-    print("Executing the following test script: " + testScript)
-    
-    parseResult = myTBTAF.parseScript(testScript)
+    if len(sys.argv) > 1:
+        if sys.argv[1] == '-interactive':
```

Comment:
before comparison, please transform the argument to lower case to ensure proper comparison